diff --git a/dist/index.mjs b/dist/index.mjs
index 874bd030155252da6a8cb70846c6d054bac5c93e..98e46ae94b2c2860dbd7c225af8d3facda6cf15c 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1,5 +1,7 @@
 import { randomUUID } from "node:crypto";
-import { decode, sign, verify } from "jsonwebtoken";
+import jwt from "jsonwebtoken";
+
+const { decode, sign, verify } = jwt;
 
 //#region src/jwt-config.ts
 const supportedAlgorithms = new Set([
